### PR TITLE
Update stripe.md

### PR DESCRIPTION
**Pull request by NifemiOmotola**
Modified the description

## PR Description

TBD

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/pbc/all/payment-service-provider/202311.0/base-shop/third-party-integrations/stripe/stripe.md [hunk 1]
```diff
@@ -15,40 +15,51 @@ The Stripe integration in Spryker is a part of the App Composition Platform and
 ## Supported business models
 The Stripe App supports the business-to-business (B2B) and business-to-consumer (B2C) models.
 
-## Prerequisites for using the Stripe app
+## Features
+The Stripe App currently has the following features:
+1. Interface within Spryker ACP Catalog to connect with Stripe: Users can easily connect to Stripe from the App Composition Platform.
+2. Easy switch between Test & Live mode: Users have the option to test payments in either mode.
+3. Responsive Redirect Payment Page: Once a connection is set up between Spryker & Stripe, End users checking out are redirected to a Spryker-hosted payment page where they can view activated payment methods. This works both on web and mobile.
```
Comment (lenadoc) [inline docs/pbc/all/payment-service-provider/202311.0/base-shop/third-party-integrations/stripe/stripe.md:22]: do you mean that upon check out users are redirected to a Spryker-hosted payment page where they can view activated payment methods?

File: docs/pbc/all/payment-service-provider/202311.0/base-shop/third-party-integrations/stripe/stripe.md [hunk 2]
```diff
@@ -15,40 +15,51 @@ The Stripe integration in Spryker is a part of the App Composition Platform and
...
+4. View Payment Methods activated in your Stripe Dashboard
```
Comment (lenadoc) [inline docs/pbc/all/payment-service-provider/202311.0/base-shop/third-party-integrations/stripe/stripe.md:23]: don't quite get what this means. What is the "View payment methods"?

**Review by lenadoc — CHANGES_REQUESTED** [review]

File: docs/pbc/all/payment-service-provider/202311.0/base-shop/third-party-integrations/stripe/stripe.md (hunk 1 above)
Comment (NifemiOmotola) [reply docs/pbc/all/payment-service-provider/202311.0/base-shop/third-party-integrations/stripe/stripe.md:22]: yes that's correct

File: docs/pbc/all/payment-service-provider/202311.0/base-shop/third-party-integrations/stripe/stripe.md (hunk 2 above)
Comment (NifemiOmotola) [reply docs/pbc/all/payment-service-provider/202311.0/base-shop/third-party-integrations/stripe/stripe.md:23]: It means that payment emthods activated within Stripe can be viewed on the redirect payment page

**Comment by lenadoc** [comment]
Merged in https://github.com/spryker/spryker-docs/pull/2500

**Pull request closed without merging**